### PR TITLE
Fixes issue #727

### PR DESCRIPTION
**Pull request by nerd0**
- Adding configuration option for user defined namespace for coordinator
- Added a bootstrap script for prometheus docker stack

### CODE REVIEW

File: examples/prometheus_integration/bootstrap_db.sh [hunk 1]
```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
```
Comment (nerd0) [inline examples/prometheus_integration/bootstrap_db.sh]: @prateek would wiring this test within semaphore be overkill. Essentially it's testing Prometheus with M3DB as a remote storage node.

File: src/coordinator/services/m3coordinator/server/server.go [hunk 2]
```diff
@@ -174,6 +170,10 @@ func Run(runOpts RunOptions) {
 
 func setupStorages(logger *zap.Logger, session client.Session, cfg config.Configuration) (storage.Storage, func()) {
 	cleanup := func() {}
+	namespace := "metrics"
```
Comment (prateek) [inline src/coordinator/services/m3coordinator/server/server.go]: nit: use a const for the default value

File: examples/prometheus_integration/bootstrap_db.sh (hunk 1 above)
Comment (prateek) [reply examples/prometheus_integration/bootstrap_db.sh]: Only concern with adding is impact on build times. Can you see how much time build takes if you add to the existing "Docker Integration Test" job?

File: src/coordinator/services/m3coordinator/server/server.go (hunk 2 above)
Comment (nerd0) [reply src/coordinator/services/m3coordinator/server/server.go]: good call, will update

File: examples/prometheus_integration/bootstrap_db.sh (hunk 1 above)
Comment (nerd0) [reply examples/prometheus_integration/bootstrap_db.sh]: Currently the "Docker Integration Test" takes around 4 and half minutes to fully run. The longest test we have is around 10 minutes.

Comment (nerd0) [reply examples/prometheus_integration/bootstrap_db.sh]: @prateek sorry for the delay, with both the docker integration test and the prometheus test the [build](https://semaphoreci.com/m3db/m3db/branches/nerd0-issue-727/builds/8) clocks in around 7 minutes which is still under the 10 minutes builds. What do you think?

Comment (prateek) [reply examples/prometheus_integration/bootstrap_db.sh]: Nice! I'm good with adding it if you are :)

**Pull request merged**